polling_stations/apps/data_finder/helpers.py

```diff
@@ -209,6 +209,19 @@ def get_explanations(self):
                     })
         return explanations
 
+    def get_metadata(self):
+        if not self.request_success:
+            return None
+        if len(self.elections) > 0:
+            for election in self.elections:
+                if not 'metadata' in election:
+                    continue
+                if not election['metadata']:
+                    continue
+                if election['group_type'] == 'organisation':
```

Comment:
on reflection, I think its important that each ballot id has the meta record attached to it in EE - revisit this.